index.html

```diff
@@ -8,6 +8,21 @@
     <link rel="stylesheet" href="style.css">
 </head>
 <body>
+    <center>
+        <section id="header">
+            <img class ="image_geometry_1" src="images/image_geometry_1.png"/>
+           
+            <img class ="ic_menu" src="images/ic_menu.png"/>
```

Comment:
- [ ] Kindly wrap your hamburger in a` nav `tag to increase the accessibility of your website because the hamburger will contain yourr navigation elements and wrapping it in a `nav` tag will make it easier for screen readers and search engines to understand what the section is all about 👍🏽